driver/examples/src/test/resources/projects/dokka/ideprobe.conf

```diff
@@ -0,0 +1,9 @@
+include required(classpath("ideprobe.conf"))
+
+workspace {
+  path = "https://github.com/Kotlin/dokka"
+}
+
+test {
+  modules = ["ws"]
```

Comment:
can we list (in all files) more modules, preferably meaningful names that someone could think that come from the library mentioned in workspace path?